runtime/autoload/health/provider.vim

```diff
@@ -53,7 +53,7 @@ function! s:check_clipboard() abort
   if empty(clipboard_tool)
     call health#report_warn(
           \ "No clipboard tool found. Using the system clipboard won't work.",
-          \ ['See ":help clipboard"'])
+          \ ['See |clipboard|.'])
```

Comment:
Does this look good? https://github.com/neovim/neovim/pull/5557/commits/a4c6f5931cdd4e5d5edb6dd7a204b6401bb43bc9